packages/gestalt/src/Contents.js

```diff
@@ -91,15 +91,17 @@ class Contents extends Component<Props, State> {
   };
 
   componentDidMount() {
-    const { onResize, onKeyDown } = this.props;
+    const { onResize, onKeyDown, shouldFocus } = this.props;
     const { popoverRef } = this.state;
 
-    setTimeout(() => {
-      if (this.props.shouldFocus && popoverRef) {
+    function draw() {
```

Comment:
I'm not sure I understand the name of this function. What is being drawn? It seems like something along the lines of `focusPopoverRef` would be more accurate?

Reply:
Makes sense. I think this new name communicates better as the use of requestAnimationFrame is an implementation detail — the function name should communicate _what_ the function is doing, not _when_ it gets called.